src/guide/custom-directive.md

```diff
@@ -277,8 +277,50 @@ Vue.directive('my-directive', {
 
 Use this wisely though, because in general you want to avoid side-effects in your templates.
 
+### terminal
+
+In some cases, we may want to used as the **terminal** directives. For example, you may want to perform custom directive than normal directive like `v-if` or `v-for`. If you want to do so, you need to pass in `terminal: true` in your directive definition.
```

Comment:
Can't get it ) What is terminal directive? How is it different from non-terminal one? Why would I want to make a terminal? 
Tbh I don't know myself what it is and can't understand it from the text )